pyproject.toml

```diff
@@ -1,24 +1,21 @@
-[build-system]
-#a list of packages that are needed to build your package.
-requires = ["hatchling"]
-#name of the Python object that frontends will use to perform the build.
-build-backend = "hatchling.build"
+[tool.poetry]
+  name = "pieces_os_client"
+  version = "1.2.3"

```

Comment:
this is very quickly going to jump to 1.2.4 today/tomorrow with some very minor noncritical updates, just a heads up . this will also get all of the sdks to the same version number for the time being